commit cd38df37e8a03efd044e21b40c648a69b8e525ee (refs/changes/45/1645/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-03-09T16:48:34-08:00 (2 years, 7 months ago)
    
    IMGAPI-622 avoid restify's uncaughtException handler so we can get cores

diff --git a/lib/app.js b/lib/app.js
index 4af886f..4aaae72 100644
--- a/lib/app.js
+++ b/lib/app.js
@@ -307,6 +307,7 @@ function App(config, log) {
         formatters: {
             'application/json': formatJSON
         },
+        handleUncaughtExceptions: false,
         version: ['1.0.0', '2.0.0']
     });
 
@@ -405,10 +406,6 @@ function App(config, log) {
             body: body
         })(req, res, route, err);
     });
-    server.on('uncaughtException', function (req, res, route, err) {
-        req.log.error(err);
-        res.send(err);
-    });
 
     var reqAuth = getAuthMiddleware(this, config, false);
     var reqPassiveAuth = getAuthMiddleware(this, config, true);
